src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -0,0 +1,263 @@
+
+    # Set the values of fipsCode in pal_records_a and b to make them adjacent.
+    # 20063955100 and 20063955200 respectively
+    pal_records = makePalRecordsConsistent([pal_record_a, pal_record_b],
+                                           pal_low_frequency, pal_high_frequency,
+                                           device_a['userId'])
+    
+    # Set the locations of devices to reside with in service area
+    device_a['installationParam']['latitude'], device_a['installationParam'][
+        'longitude'] = 39.0373, -100.4184
+    device_b['installationParam']['latitude'], device_b['installationParam'][
+        'longitude'] = 39.0378, -100.4785
+
+    # Set the AntennaGain and EIRP capability
+    device_a['installationParam']['eirpCapability'] = 30
+    device_b['installationParam']['eirpCapability'] = 47
+    device_a['installationParam']['antennaGain'] = 16
+    device_b['installationParam']['antennaGain'] = 16
+
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam'],
+        'measCapability': device_b['measCapability']
+    }
+    conditionals = {'registrationData': [conditionals_b]}
+    del device_b['installationParam']
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+    del device_b['measCapability']
+
+    # Create the actual config.
+    devices = [device_a, device_b]
+    config = {
+        'registrationRequests': devices,
+        'conditionalRegistrationData': conditionals,
+        'palRecords': pal_records,
+        'ituDataPath': CONFIG.GetItuDir(),
+        'terrainDataPath': CONFIG.GetTerrainDir(),
+        'landCoverDataPath': CONFIG.GetLandCoverDir(),
+        'serverCert': os.path.join('certs/sas.cert'),
+        'serverKey': os.path.join('certs/sas.key')
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_PCR_1_default_config)
+  def test_WINNF_FT_S_PCR_1(self, config_filename):
+    """ Successful Maximum PPA Creation
+
+    Checks PPA generated by SAS UUT shall be fully contained within the service area.
+    """
+    # Load the Config file
+    config = loadConfig(config_filename)
+    pal_records = config['palRecords']
+    itu_data_path = config['ituDataPath']
+    terrain_data_path = config['terrainDataPath']
+    landcover_data_path = config['landCoverDataPath']
+
+    # light checking of itu,terrain and landcover data path exists
+    self.assertTrue(os.path.exists(itu_data_path), msg='ITU Data path is not configured')
+    self.assertTrue(os.path.exists(terrain_data_path), msg='Terrain Data path is not configured')
+    self.assertTrue(os.path.exists(landcover_data_path), msg='LandCover Data path is not configured')
+
+    # Whitelist user IDs.
+    for device in config['registrationRequests']:
```

Comment:
Not needed, assertRegistered will do this.

Reply:
We have   removed the injectFCCId and injectUserID API from testcase.